docs/copied-from-beats/outputconfig.asciidoc

```diff
@@ -254,11 +258,13 @@ Example elasticsearch output with `pipelines`:
 
 ["source","yaml"]
 ------------------------------------------------------------------------------
-filebeat.prospectors:
-- paths: ["/var/log/app/normal/*.log"]
+filebeat.inputs:
```

Comment:
is this relevant to us?

Reply:
no, and I checked - it doesn't show up in the rendered docs